package book.bible2nd.ch8Exceptions.part1;

public class ExceptionEx7 {
    public static void main(String[] args) {
//        throw new Exception();    //Exception을 강제로 발생(오류발생!)(컴파일도 안됨)
    }
}
